persistence-cassandra-3.11/src/test/java/io/stargate/db/cassandra/impl/CassandraPersistenceIT.java

```diff
@@ -36,6 +35,7 @@
   @BeforeAll
   public static void createPersistence(ClusterConnectionInfo backend) throws IOException {
     baseDir = Files.createTempDirectory("stargate-cassandra-3.11-test").toFile();
+    baseDir.deleteOnExit();
```

Comment:
This may not delete non-empty dirs, AFAIK.

Reply:
Javadoc doesn't say anything about non-empty so it _should_ delete them.